addons/mrp/static/src/js/report_stock_forecasted.js

```diff
@@ -0,0 +1,54 @@
+odoo.define('mrp.ReplenishReport', function (require) {
+"use strict";
+
+const replenishReport = require('stock.ReplenishReport');
+const core = require('web.core');
+
+const MRPReplenishReport = replenishReport.extend({
+
+    _bindAdditionalActionHandlers: function () {
+        this._super();
+        let rr = this.$el.find('iframe').contents().find('.o_report_replenishment');
+        rr.on('click', '.o_report_replenish_change_production_priority', this._onClickChangeProductionPriority.bind(this));
+        rr.on('click', '.o_report_replenish_unreserve_production', this._onClickUnreserveProduction.bind(this));
+    },
+
+    /**
+     * Change the priority of the specified production, then reload this report.
+     *
+     * @returns {Promise}
+     */
+    _onClickChangeProductionPriority: function(ev) {
+        const productionId = parseInt(ev.target.parentNode.getAttribute('production-id'));
+        this._rpc( {
+            model: 'mrp.production',
+            args: [[productionId]],
+            method: 'do_change_priority'
+        }).then((result) => {
+            return this._reloadReport();
+        });
+    },
+
+     /**
+     * Unreserve the specified production, then reload this report.
+     *
+     * @returns {Promise}
+     */
+    _onClickUnreserveProduction: function(ev) {
+        const productionId = parseInt(ev.target.getAttribute('production-id'));
+        this._rpc( {
+            model: 'mrp.production',
+            args: [[productionId]],
+            method: 'do_unreserve'
+        }).then((result) => {
+            return this._reloadReport();
+        });
+    }
```

Comment:
You could probably write the model on the ev.target to avoid overriding this method. And instead do:
```js
 const productionId = parseInt(ev.target.getAttribute('production-id'));
 const model = ev.target.getAttribute('model'));
 const value = check event target class (si fa-star-o alors 1 sinon 0)
        this._rpc( {
            model: 'model,
            args: [[productionId], {priority: value}],
            method: 'write'

        }).then((result) => {
            return this._reloadReport();
        });
```